src/components/structures/MatrixChat.js

```diff
@@ -927,6 +927,9 @@ export default React.createClass({
     },
 
     _viewHome: function() {
+        this.setStateForNewView({
```

Comment:
I think it would be a bit clearer if we had a comment here to clarify this is needed because home page requires the `LOGGED_IN` view and it might not be set already if navigate directly to /home via the URL.